crm-project/src/main/java/CRM/utils/Validations.java

```diff
@@ -113,8 +113,21 @@ public static void validateCreatedItem(ItemRequest item) {
             item.setDescription("");
     }
 
-    public static void validateCreatedComment(Comment comment) {
-        // validate each field of the comment using validate(regex, field)
+    public static void validateCreatedComment(CommentRequest comment) {
+        // validate each field of the item using validate(regex, field)
+        try {
+            validate(comment.getParentItemId(), Regex.ID.getRegex());
+        } catch (NullPointerException e){
+            logger.warn("Parent item ID is null");
+        }
```

Comment:
Unlike item, comment can not have parentId as null... it must be created under an item thus must have parentId